Shared/dugga.js

```diff
@@ -183,13 +183,22 @@ function htmlEntities(str) {
 // AJAX Service: Generic AJAX Calling Function with Prepared Parameters
 //----------------------------------------------------------------------------------
 
-function AJAXService(opt,apara,kind)
+function AJAXService(opt,apara,kind)	
 {
 	var para="";
 	for (var key in apara) {
-			para+="&"+key+"="+encodeURIComponent(htmlEntities(apara[key]));
-	}
-				
+			// Handles all the individual elements in an array and adds the array as such: &key=val1,val2,val3
+			if (apara[key].constructor === Array) {
```

Comment:
If a user opens the EditorV50.php, an error message is displayed. The message probably has something to do with this line.
![errordump2](https://cloud.githubusercontent.com/assets/11715509/7136561/5d9111e8-e2b2-11e4-87cb-3cd76f92657f.png)